A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Misc.xml

```diff
@@ -294,4 +294,56 @@
             <TOOLTIPID>%((A:APU PCT RPM, Percent) 0 &gt; (A:APU PCT RPM, Percent) #AVAIL_PERCENTAGE# &lt; and)%{if}TT:COCKPIT.TOOLTIPS.UNAVAILABLE%{end} %((A:APU PCT RPM, Percent) #AVAIL_PERCENTAGE# &gt;= (A:APU SWITCH, Bool) and)%{if}TT:COCKPIT.TOOLTIPS.APU_STARTER_TURN_OFF%{else}TT:COCKPIT.TOOLTIPS.APU_STARTER_TURN_ON%{end}</TOOLTIPID>
         </UseTemplate>
     </Template>
+        <Template Name="FBW_ENGINE_Switch_Master_SubTemplate">
+		<DefaultTemplateParameters>
+			<NODE_ID>ENGINE_Switch_Master_#ID#</NODE_ID>
+			<ANIM_NAME>ENGINE_Switch_Master_#ID#</ANIM_NAME>
+			<WWISE_EVENT_1>engine_master_switch_on</WWISE_EVENT_1> 
+			<WWISE_EVENT_2>engine_master_switch_off</WWISE_EVENT_2> 
+			<PART_ID>ENGINE_Switch_Master</PART_ID>
+			<Switch Param="TYPE">
+				<Case Value="LOCKED">
+					<LOCK_NODE_ID>ENGINE_Switch_Master_#ID#_Lock</NODE_ID>
+					<LOCK_ANIM_NAME>ENGINE_Switch_Master_#ID#_Lock</ANIM_NAME>
+					<VAR_SCOPE>I</VAR_SCOPE> <!-- I or L -->
+				</Case>
+			</Switch>
+			<ANIMTIP_0>TT:COCKPIT.TOOLTIPS.ENGINE#ID#_MASTER_TURN_ON</ANIMTIP_0>
+			<ANIMTIP_1>TT:COCKPIT.TOOLTIPS.ENGINE#ID#_MASTER_TURN_OFF</ANIMTIP_1>
+		</DefaultTemplateParameters>
+				<Component ID="#NODE_ID#" Node="#NODE_ID#">
+					<Update Frequency="5">
```

Comment:
Why did you change this update tag? in the original, this is Once="True" instead of Frequency="5"

Reply:
The key event fired by the quadrant actually didn't execute the code of this switch and this toggles the starter after the "toggle fuel valve" key is sent.
 I agree it's a rather bad workaround but I don't see how I can trigger the switch directly as it only seems to respond to "mouse left clicks". 
I tried using a two state switch but was not sure if this would change something. 

If you have more ideas, I'd be happy to hear!